crates/solvers/src/infra/config/dex/file.rs

```diff
@@ -64,6 +77,18 @@ fn default_smallest_partial_fill() -> eth::U256 {
     eth::U256::exp10(16) // 0.01 ETH
 }
 
+fn default_back_off_growth_factor() -> f64 {
+    1.0
+}
+
+fn default_min_back_off() -> Duration {
+    Duration::default()
+}
+
+fn default_max_back_off() -> Duration {
+    Duration::default()
```

Comment:
TBD: Turning it off by default.

Reply:
Can we use the same default values that we already have for the rate limiter in the orderbook/autopilot crate?

Reply:
The default for the shared rate limiter is https://github.com/cowprotocol/services/blob/33d1a1eb879a0031e8a10a799d21cab4809e336d/crates/shared/src/rate_limiter.rs#L43 which is the most transparent to be turned off by default.